packages/venia-ui/lib/components/AccountMenu/accountMenuItems.css

```diff
@@ -1,3 +1,11 @@
+.root {
+    display: grid;
+    grid-auto-flow: row;
+    justify-content: stretch;
+    row-gap: 1.5rem;
+    padding: 1.5rem 2.5rem;
```

Comment:
This is affecting the account menu within left nav.

[![Image from Gyazo](https://i.gyazo.com/a205cec22f153e8d2d264a3dd6e5eb8d.png)](https://gyazo.com/a205cec22f153e8d2d264a3dd6e5eb8d)